### DISCUSSION TITLE
i think maybe use frame replace the page comment will be better.

### DISCUSSION THREAD

**Discussion by q2333gh** [cmu-db/bustub#353] [Ideas]
![5078@ %YW8BWKW{{XIBJ8K](https://user-images.githubusercontent.com/32679742/190545743-4a9a6744-2c70-4237-91c5-42d3b59bcf32.png)
since the data is in the memory,so is it better to name it frame?

![90fa2363073eeb60279745192f7a7d0aa840206da1e52022780a5a015be698f5](https://user-images.githubusercontent.com/32679742/190545990-e7e6516e-39fb-4c1c-a6e2-c7db6e698eb0.png)

**Comment by skyzh** [comment]
People generally want logically-consistent API convention. FetchPage uses page_id, so UnpinPage also uses page_id, in my opinion.
